### PR TITLE
Create MainContent component that renders blocks

### PR DESCRIPTION
**Pull request by jtart**
_Creates a component that renders blocks from the articles data to string. It renders the data for the headline into the headline component, and everything else into a paragraph element. This is a pre-cursor for rendering each individual block into a React component in their own right. It does not integrate with the Article component._

* [x] Fixes https://github.com/bbc/simorgh/issues/45
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* [x] Tests added for new features
* [x] Tests pass - `npm test`
* [x] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [x] Test engineer approval

### CODE REVIEW

File: src/app/components/Article.jsx [hunk 1]
```diff
@@ -10,7 +11,36 @@ const Headline = styled.h1`
   font-size: 2em;
 `;
 
+const renderModel = ({ blocks }) =>
```
Comment (jamesbhobbs) [inline src/app/components/Article.jsx]: Suggest moving this to another file while it's easy to do so.

File: src/app/components/Article.jsx [hunk 2]
```diff
@@ -34,13 +64,16 @@ class Article extends Component {
...
+
+    const renderedModel = renderModel(model);
     return (
       <Fragment>
         <Helmet htmlAttributes={{ lang: 'en-GB' }}>
           <title>{headline}</title>
         </Helmet>
         <Header />
-        <Headline>{headline}</Headline>
+        {renderedModel}
```
Comment (jamesbhobbs) [inline src/app/components/Article.jsx]: Suggest renaming to something like articleBody

File: src/app/components/Article.jsx [hunk 3]
```diff
@@ -10,7 +11,36 @@ const Headline = styled.h1`
   font-size: 2em;
 `;
 
+const renderModel = ({ blocks }) =>
+  blocks.map(block => {
+    const { type, blockId } = block;
+    const blockString = JSON.stringify(block);
+
+    let Element = 'p';
```
Comment (jamesbhobbs) [inline src/app/components/Article.jsx]: This is fine at the moment. Will be interesting to see how it develops as it gets significantly more convoluted.

File: src/app/components/Article.jsx [hunk 4]
```diff
@@ -24,8 +54,8 @@ class Article extends Component {
 
     try {
       const response = await fetch(url);
-      const data = await response.json();
-      return { data };
+      const { model } = await response.json();
```
Comment (jamesbhobbs) [inline src/app/components/Article.jsx]: I presume you've left this as scenario-01 to a) keep this PR small and b) not start with a more complex example. However we should probably have an issue that we need to iterate the logic above until it works on our production examples.

Furthermore we need an issue to remove this hardcoding.

Given that this PR starts that stream of work I suggest these and other related issues are written up now.

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]

File: src/app/components/MainContent/index.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+storiesOf('MainContent', module).add('with just a headline', () => {
+  const blocksOnlyHeadline = data.blocks.filter(
+    ({ type }) => type === 'headline',
+  );
+  const dataOnlyHeadline = { blocks: blocksOnlyHeadline };
+
+  return <MainContent data={dataOnlyHeadline} />;
+});
+
+storiesOf('MainContent', module).add('with a headline and other blocks', () => (
```
Comment (jamesbhobbs) [inline src/app/components/MainContent/index.stories.jsx]: No need to have this separate, see docs https://storybook.js.org/basics/writing-stories/#writing-stories

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]

**Review by jamesbhobbs — DISMISSED** [review]

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

**Pull request merged**